packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.33.3"
+  changes:
+    - description: Fix Redis metric field type.
```

Make the changelog entry description a bit more descriptive.

```suggestion
    - description: Fix Redis metric type for 'persistence.rdb.bgsave_in_progress'. Metric type should be boolean instead of long.
```

---

Done